frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -60,7 +61,19 @@ export const RequestedChangesDisplay = ({
         data.some((responseItem) => responseItem.message);
 
     function handleSaveToCsvClick() {
-        return saveToCsv(data, {
+        const dataWithErrorMessage = data.map((item) => {
+            return {
+                ...item,
+                errorMessageDetails: renderToString(
+                    <ValidationErrorMessage
+                        errorCode={item.errorCode}
+                        field={item.field}
+                        message={item.message}
+                    />
+                ).replace(/<(.|\n)*?>/g, ""),
```

Comment:
Do we need the replace here? Can the logic be in the ValidationErrorMessage itself? Seems like it only needs to be applied if the ValidationErrorMessage returns the `message` from the backend.

Reply:
We'll still want the markup to be replaced by the ValidationErrorMessage component in other circumstances.  This is purely for making a "plaintext"/sanitized version for the downloaded CSV file so there isn't any unnecessary markup there.

Reply:
Yeah, since we need to stringify the component to render the text in the CSV, I used the 2nd option from [this list of HTML tag Regexs](https://www.pagecolumn.com/tool/all_about_html_tags.htm) to remove them. Is this a no-op or shall I add a comment to this code explaining what's happening?

Reply:
Hmmm good point -- let's add an inline comment just for our future selves!